DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Centre/Administrator/SearchableAdminViewModelTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Centre.Administrator
+{
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Administrator;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class SearchableAdminViewModelTests
+    {
+        
+        [Test]
+        public void SearchableAdmin_centre_manager_should_not_be_able_to_deactivate_their_own_account()
+        {
+            // Given
+            var adminUser = UserTestHelper.GetDefaultAdminUser(id: 1, isCentreAdmin: true);

```

Comment:
You want `isCentreManager` rather than `isCentreAdmin`. CentreAdmin = anyone who can use the tracking system. Same elsewhere in this file.

Reply:
Changed to isCentreManager

Reply:
Still need to do this elsewhere in the file